routing.asciidoc

```diff
@@ -82,304 +99,201 @@ As Wei gets message from the escrow that Bob has deposited the 10 golden coins W
+As we'll see in the later chapters, given a set of adequate cryptographic constructs, many other instantiations are possible as well.
+
+Before we dive into the specifics of HTLCs, it may be helpful to first build intuition on an abstraction over this concrete concept.
+First, let's unpack what it means for something to be a conditional chained end to end secure payment:
+
+== Conditional End to End Secure Payments by Construction ==
+
+=== Conditional Payments ===
+
+A payment can be said to be conditional, if the completion of the payment relies on the completion of a certain event. 
+In the golden coins example, this "condition" was the reveal of a hash pre-image. 
+We could feasibly substitute this hash pre-image reveal for any other construct with "hardness" properties. Namely: it should be infeasible for a party that doesn't know the proper "solution" of the condition to satisfy it, the "description" of the condition shouldn't give away any information about the true "solution", and once a solution has been chosen and a description created from it, it shouldn't be possible to "alter" that solution and have it still be a valid condition for the description. 
+
+The payment should _only_ be able to be redeemed if a valid solution is revealed. Critical, all conditions need to be timed in order to allow the construct to return the funds back to the sender if a solution to this condition isn't revealed.
+The combination of the condition, and a timeout on the condition gives the payment a trait we commonly refer to as atomicity: either the payment happens, or the receiver if refunded the funds.
+
+=== Conditional Chained Payment ===
+
+Building upon our conditional payment, it may be possible to *chain* this payment, allowing it to involve the payer, the payee, and possibly several intermediaries. 
+Each intermediary, is able to present a _slightly_ modified version of the condition (without invalidating it all together), and so so in an iterated manner until the conditional payment reaches the payee.
+Once it reaches the payee, then the payment should be able to be _iteratively_ resolved, starting at the payee all the way back to the payer. 
+
+Each chaining creates an "incoming" and "outgoing" conditional payment.
+A node receives a conditional payment from a party (incoming condition), and then extends the conditional payment to the next party in the chain (outgoing condition).
+The payment is extended in from payer to payee, but settled from payee to payer, as each of the intermediaries gain the solution to the outgoing condition, and use that (possibly augmenting it) to satisfy the incoming solution.
+
+Typically the payer rewards the intermediaries by sending slightly more than the payment amount, in order to allow the intermediaries to send out less with their outgoing payment than what they received from the incoming payment.
```

Comment:
maybe this and the next sentence as an info box as it is a relevant idea but not necessary to understand conditional chained payments

Reply:
IMO it's a necessary piece of information for the "chained" aspects, as it's important that the incentives are built in and outlined.